internal/services/servicebus/servicebus_namespace_disaster_recovery_config_data_source_test.go

```diff
@@ -22,7 +22,7 @@ func TestAccDataSourceServiceBusNamespaceDisasterRecoveryConfig_basic(t *testing
 			Config: r.basic(data),
 			Check: acceptance.ComposeTestCheckFunc(
 				check.That(data.ResourceName).Key("name").Exists(),
-				check.That(data.ResourceName).Key("resource_group_name").Exists(),
+				check.That(data.ResourceName).Key("namespace_id").Exists(),
```

Comment:
add similar `if !features` block to still test `resource_group_name`?

Reply:
Nah, this is fine to leave as is